### PR TITLE
fix: improve LGBM error message for invalid slot names

### PR DESCRIPTION
**Pull request by mhamilton723**

### CODE REVIEW

**Review by imatiach-msft — APPROVED** [review]

File: lightgbm/src/test/scala/com/microsoft/ml/spark/lightgbm/split2/VerifyLightGBMRegressor.scala [hunk 1]
```diff
@@ -41,6 +42,20 @@ class VerifyLightGBMRegressor extends Benchmarks
     super.testSerialization()
   }
 
+  test("foo") {
```
Comment (imatiach-msft) [inline lightgbm/src/test/scala/com/microsoft/ml/spark/lightgbm/split2/VerifyLightGBMRegressor.scala:45]: remove?

Comment (mhamilton723) [reply lightgbm/src/test/scala/com/microsoft/ml/spark/lightgbm/split2/VerifyLightGBMRegressor.scala]: TY!

**Pull request merged**